### DISCUSSION TITLE
`Octo review resume` makes me think there should be an `Octo review pause` - should there be a corresponding `pause` command?

### DISCUSSION THREAD

**Discussion by macintacos** [pwntester/octo.nvim#356] [Ideas]
Was testing out the PR reviewing functionality (which is great!) and saw that there was an `Octo review resume` command, which is great - however, it's not immediately obvious how one is supposed to "pause" the review in order for it to become resumable.

How is one expected to "pause" a review, and would it be possible to have this added as a command (even if it's just a simple convenience for the user)?

**Comment by pwntester** [comment]
Thanks for the kind words. There is no way to pause a review. If you start one in the web UI you will see that it remains in "pending" state until you submit it or cancel it. Even if you close the tab, when you open that PR again you will see that the pending comments are still there. In Octo, this "resume" is not automatic. If you open a PR with pending comments in Octo, it will not load that pending review, you have to do it explictly by calling the "resume" command.
